lib/game.rb

```diff
@@ -0,0 +1,75 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]
+  ]
+
+  def initialize(*args)
+    @player_1 = args[0] || Player::Human.new("X")
+    @player_2 = args[1] || Player::Human.new("O")
+    @board = args[2] || Board.new
+  end
+
+  def current_player
+    @board.turn_count.even? ? @player_1 : @player_2
+  end
+
+  def draw?
+    ( @board.full? && !won? ) ? true : false
+  end
```

Comment:
Use the implicit boolean return value of the expression instead of adding the ternary. 

```ruby
def draw?
  @board.full? && !won?
end
```